packages/v4/src/content/accessibility/progress-stepper/progress-stepper.md

```diff
@@ -0,0 +1,19 @@
+---
+id: Progress stepper
+section: components
+---
+
+A **progress stepper** is used to provide information to the user about the state of their current progress on a multistep interaction.
+
+**Keyboard users** should be able to use **Tab** and **Shift + Tab** navigate to and between progress steps with popovers. Keyboard users should 
+be able to open and close the popover on a progress step using **Enter** or **Space**.
+
+**Screen reader users** should be able to discern the label, status, and optional description of every progress step. 
+The aria-label provided to each step should indicate all information being communicated by the progress step's icon, including the variant and the completed status.
+Screen reader users should also be able to discern whether a step has an associated popover they can trigger and the contents of that popover.
```

Comment:
The previous line is also tied into this comment, but I wanted to keep this separate from the comments already shared on that one.

Could lines 11-13 of the file for screen reader users be linked more to a single thought, rather than sort of coming off as somewhat separate ones? Roughly something along the lines of:

```suggestion
**Screen reader users** should be able to discern the label, status, and optional description of every progress step via the `aria-label` prop. This should include all information being communicated by the progress step's icon, such as the variant and the completed status, as well as whether the step has an associated popover they can trigger. Screen reader users should also be able to access the contents of any popover associated with a step.
```

Of course taking into account if any changes are made based on the comments above.